src/components/Game.jsx

```diff
@@ -38,19 +39,22 @@ class Game extends Component {
 
     const mergeWordList = Object.assign({}, this.state.wordList)
 
-    if (!mergeWordList[word]) {
+    if ( word.length > 0 && !mergeWordList[word] ) {
```

Comment:
Yes, if it's blank, we don't want to add it to the list.  The requirement didnt say min length of 2. If length is 3, the point is 1. I assume that length between 1 and 3 has a score of 0.